docs/dev/table/sql/gettingStarted.md

```diff
@@ -0,0 +1,226 @@
+<!-- #### Processing mode (append, changelog, ..)-->
+
+### SQL Client result modes
+The CLI supports **three modes** for maintaining and visualizing results.
+
+The **table mode** materializes results in memory and visualizes them in a regular, paginated table representation. It can be enabled by executing the following command in the CLI:
+
+{% highlight text %}
+SET execution.result-mode=table;
+{% endhighlight %}
+
+The **changelog mode** does not materialize results and visualizes the result stream that is produced by a [continuous query](streaming/dynamic_tables.html#continuous-queries) consisting of insertions (`+`) and retractions (`-`).
+
+{% highlight text %}
+SET execution.result-mode=changelog;
+{% endhighlight %}
+
+The **tableau mode** is more like a traditional way which will display the results on the screen directly with a tableau format.
+The displaying content will be influenced by the query execution type(`execution.type`).
+
+{% highlight text %}
+SET execution.result-mode=tableau;
+{% endhighlight %}
+
+Note that when using **tableau mode** with the streaming query, the result will be continuously printed on the console. If the input data of 
+this query is bounded, the job will terminate after Flink processed all input data, and the printing will also be stopped automatically.
+Otherwise, if you want to terminate a running query, just type `CTRL-C` in this case, the job and the printing will be stopped.
+
+## Dynamic tables and continuous queries
+Standard SQL has not been designed with streaming data in mind. As a consequence, there are few conceptual gaps between standard SQL and stream processing. SQL is designed for finite or data-at-rest processing. SQL queries are completed after processing the fixed size of the input data. On the contrary, streaming queries are designed for an infinite amount of data. Therefore, they continue execution forever as queries do not have access to all the data. 
+
+A [continuous query](/table/streaming/dynamic_tables.html#continuous-queries) never terminates and produces a dynamic table as result. [Dynamic tables](/table/streaming/dynamic_tables.html) are the core concept of Flink’s Table API and SQL support for streaming data. 
+
+Aggregations on the continuous stream need to store aggregated results continuously during the execution of the query. For example, if the number of employees needs to be counted for each department from the data stream, the aggregated count needs to be stored for each department during the execution of the query. 
+
+ {% highlight sql %}
+ SELECT count(1) as EmpCount, DeptId from EmployeeTableSource GROUP BY DeptId;
+ {% endhighlight %} 
+
+Such queries or operators where the state needs to store are referred to as materialized operators. A materialized view caches the result of the query such that the query does not need to be evaluated when the view is accessed. During the execution of the query, the state can grow with every new record processed. 
```

Comment:
```suggestion
Such queries, where state needs to be stored, are referred to as _stateful_. Under the hood, this is similar to a materialized view that caches the result of the query such that the query doesn't need to be evaluated when the view is accessed. During the execution of the query, the state can grow with every new record processed. 
```

Reply:
@morsapaes  Updated the comments but do we have any suggestion such that if we can link the materialized view?